crates/db_views/src/comment_view.rs

```diff
@@ -347,6 +347,9 @@ impl<'a> CommentQuery<'a> {
 
     query = match self.sort.unwrap_or(CommentSortType::Hot) {
       CommentSortType::Hot => query.then_order_by(comment_aggregates::hot_rank.desc()),
+      CommentSortType::Controversial => {
+        query.then_order_by(comment_aggregates::controversy_rank.desc())
```

Comment:
If we don't do the Reddit method of controversy calculation, should this add a second `ORDER BY` for activity as @dcormier suggested? Same applies for the other places that do this ordering.

Reply:
Yes I think it should also order by hot or active. Otherwise the same posts will forever stay at the top of controversial.